tsdb/index/index_test.go

```diff
@@ -558,3 +558,33 @@ func TestSymbols(t *testing.T) {
 	}
 	testutil.Ok(t, iter.Err())
 }
+
+func TestIndexWriterStageString(t *testing.T) {
```

Comment:
I think `switch ... { case: ... }` is pretty reliable and we don't require this test :)
The test could make more sense if it is general enough to check if we are missing to add string representation for any of `indexWriterStage`, which I don't think we can do easily? (and maybe not worth it).